### ISSUE TITLE
Feature request: Automatic underscores for Greek letters

### ISSUE THREAD

**Issue by Sinthoras7** [tecosaur/LaTeX-auto-activating-snippets#38]
It would be useful if the automatic underscores also worked for Greek letters, such that when there is a `\alpha` and I type `1` not `\alpha1` but `\alpha_1` appears.

(This would be useful, but TBH if this is not easily doable, I would prioritize #30 over this, because this is just a feature, which would make this package more useful/better, the other one is a bug, which makes it unusable, when working a lot with siunitx/chemical environments.)

**Comment by ymarco** [comment]
The thing is, we'd need a white/blacklist, because for things like \cdot if I write \cdot1 I definitely didn't mean \cdot_1.

I have a WIP aas branch which makes a snippet data structure and lets you tag snippets, when/if I finish it we could look at the previous macro in the buffer, check if it's a snippet and if it has a "subcriptable" property and add subscripts with that.
That adds considerable complexity though, I'm not sure it'd be worth it.

**Comment by Sinthoras7** [comment]
Yeah, I also think we'd need a whitelist/blacklist. I would probably just whitelist the Greek letters and wait, if there arise any issues, which would be solved by whitelisting more. Ideally, include a method, which users can use to add things to the whitelist

This snippet-data-structure would only work, if the Greek letter was also created by an aas-macro, wouldn't it?
Without directly checking, whether the expression before is element of the whitelist, it would be pretty tricky to expand it, no matter how it was inserted, I guess.

**Comment by ymarco** [comment]
> This snippet-data-structure would only work, if the Greek letter was also created by an aas-macro, wouldn't it?
> Without directly checking, whether the expression before is element of the whitelist, it would be pretty tricky to expand it, no matter
> how it was inserted, I guess.

Right, yeah

**Comment by Sinthoras7** [comment]
A solution for users, who use the Unicode symbols, would be to create a variable, where the user can whitelist certain chars for the auto underscores. 
Then, one could add the Unicode symbols for the Greek letters, f.e. 𝛼, 𝛽, 𝛾, 𝛿, etc..
